modules/thermalmanager.dot

```diff
@@ -0,0 +1,161 @@
+/* -*- mode: c -*-
+ *
+ * Graph of thermal measurement control flow
+ *
+ * To create PNG image from this, use:
+ *
+ * % dot -T png thermalmanager.dot -o thermalmanager.png
```

Comment:
Also, is this hand-written or generated from profiling/call graph tooling? Can it be generated with call graph tooling?